pipelines/tekton/aiedge-e2e/aiedge-e2e.pipeline.yaml

```
@@ -116,24 +124,54 @@ spec:
     description: The buildah version used to build the model container image
     value: $(tasks.retrieve-build-image-info.results.buildah-version)
   tasks:
+  - name: fetch-model-registry
+    params:
+    - name: model-name
+      value: $(params.model-name)
+    - name: model-version
+      value: $(params.model-version)
+    - name: model-registry-hostname
+      value: $(params.model-registry-hostname)
+    - name: model-registry-enabled
+      value: $(params.model-registry-enabled)
+    - name: git-model-repo
+      value: $(params.git-model-repo)
+    - name: git-model-revision
+      value: $(params.git-model-revision)
+    - name: model-dir
+      value: model_dir-$(params.model-name)/
+    - name: fetch-model
+      value: $(params.fetch-model)
+    - name: modelRelativePath
+      value: $(params.modelRelativePath)
```

Missing:
```suggestion
      value: $(params.modelRelativePath)
    - name: s3-bucket-name
      value: $(params.s3-bucket-name)
```

Also, it would be good to have the parameters in the same order both in the Pipeline here and in the Task.
Another improvement can be to have the `s3-bucket-name` parameter without the default value in the task. This way, the pipeline would most likely fail that the pipeline is not passing the parameter to the task. But once properly linked, it won't be failing because the pipeline parameter `s3-bucket-name` itself has a default value of `""`.

---

👍  agree for sure

I think this has been a persistent issue in the repo for a while. Params being out of order or the case we use to be different. I think @biswassri you can fix that here but also we can have a clean up PR for this.